docs/samples/Microsoft.ML.Samples/Trainers.cs

```diff
@@ -6,6 +6,8 @@
         using Microsoft.ML.Runtime.Data;
         using Microsoft.ML.Runtime.Learners;
         using Microsoft.ML.Trainers;
+        using Microsoft.ML.Runtime.LightGBM;
```

Comment:
>LightGBM [](start = 35, length = 8)

Something tells me what Microsoft.ML.Samples.csproj doesn't have reference to Microsoft.ML.LightGBM and Microsoft.ML.FastTree and this is main reason why your PR is blocked in build system.